core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/export/localfile/ExportToLocalFile.java

```diff
@@ -24,6 +24,7 @@
 import uk.gov.gchq.gaffer.commonutil.Required;
 
 import uk.gov.gchq.gaffer.operation.export.ExportTo;
+import uk.gov.gchq.gaffer.operation.serialisation.TypeReferenceImpl;
```

Comment:
We can import the inner class `IterableString` directly.
```suggestion
import uk.gov.gchq.gaffer.operation.serialisation.TypeReferenceImpl.IterableString;
```

Reply:
I do prefer your way but did it more for consistency, around 65 places in Gaffer do it this way but 3 the better way. I suppose doing the better change is more meaningful than the consistent one..